hack/validation/test_release.py

```diff
@@ -0,0 +1,181 @@
+
+
+def test_de_interacts_with_neighborhood_humans_with_owner(hles_dog2_data):
+    for row in hles_dog2_data:
+        assert 'de_interacts_with_neighborhood_humans_with_owner' in row
+        assert row['de_interacts_with_neighborhood_humans_with_owner'] in {'True', 'False', ''}
+
+
+def test_dd_acquired_st(hles_dog1_data):
+    found_affected_row = False
+    for row in hles_dog1_data:
+        if row['entity:dog_id_id'] in ('18069', '32705', '40519', '63438', '89055'):
+            found_affected_row = True
+            assert row['dd_acquired_state'] not in {'', 'NA'}
+
+    assert found_affected_row is True
+
+
+def test_dd_activities_obedience(hles_dog1_data):
+    found_affected_row = False
+    for row in hles_dog1_data:
+        if row['entity:dog_id_id'] in {'76099'}:
+            assert 'dd_activities_obedience' in row
+            assert row['dd_activities_obedience'] == ''
+            found_affected_row = True
+    assert found_affected_row is True
+
+
+def test_dd_activities_agility(hles_dog1_data):
+    found_affected_row = False
+    for row in hles_dog1_data:
+        if row['entity:dog_id_id'] in {'34888'}:
+            assert 'dd_activities_agility' in row
+            assert row['dd_activities_agility'] == ''
+            found_affected_row = True
+    assert found_affected_row is True
+
+
+def test_dd_activities_breeding(hles_dog1_data):
+    found_affected_row = False
+    for row in hles_dog1_data:
+        if row['entity:dog_id_id'] in {'66316'}:
+            assert 'dd_activities_breeding' in row
+            assert row['dd_activities_breeding'] == ''
+            found_affected_row = True
+    assert found_affected_row is True
+
+
+def test_dd_activities_breeding(hles_dog1_data):
+    found_affected_row = False
+    for row in hles_dog1_data:
+        if row['entity:dog_id_id'] in {'19971'}:
+            assert 'dd_activities_hunting' in row
+            assert row['dd_activities_hunting'] == ''
+            found_affected_row = True
+    assert found_affected_row is True
+
+
+def test_dd_activities_field_trials(hles_dog1_data):
+    found_affected_row = False
+    for row in hles_dog1_data:
+        if row['entity:dog_id_id'] in {'13359'}:
+            assert 'dd_activities_field_trials' in row
+            assert row['dd_activities_field_trials'] == ''
+            found_affected_row = True
+    assert found_affected_row is True
+
+
+def test_dd_activities_working(hles_dog1_data):
+    found_affected_row = False
+    for row in hles_dog1_data:
+        if row['entity:dog_id_id'] in {'82623'}:
+            assert 'dd_activities_working' in row
+            assert row['dd_activities_working'] == ''
+            found_affected_row = True
+    assert found_affected_row is True
+
+
+def test_st_portal_account_creation_date(hles_dog1_data):
+    for row in hles_dog1_data:
+        assert 'st_portal_account_creation_date' in row
+        assert len(row['st_portal_account_creation_date']) > 0
```

Comment:
```suggestion
        assert bool(row['st_portal_account_creation_date'])
```
up to you if this sacrifice in readability is acceptable